File: cash.c
#include<stdio.h>
#include<cs50.h>
#include<math.h>

int main(void)
{
    float input = get_float("Change owed: ");
    while (input < 0)
    {
        input = get_float("Change owed: ");
    }
    int coins = round(input * 100);

    int count = 0;
    while (coins >= 25)
    {
        // if still denomination with 25c change
        coins = coins - 25;
        count++;
    }
    while (coins >= 10)
    {
        // if still denomination with 10c change
        coins = coins - 10;
        count++;
    }
    while (coins >= 5)
    {
        // if still denomination with 5c change
        coins = coins - 5; 
        count++;
    }
    while (coins >= 1)
    {
        // if still denomination with 1c change
        coins = coins - 1;
        count++;
    }
    printf("%d\n", count);
}

File: cash.py
from cs50 import get_float

inf = -1.0

while inf < 0:
    inf = get_float("Change owed: ")
inf = inf * 100
inp = round(inf)
count = 0
while inp >= 25:
    count = count + 1
    inp = inp - 25
while inp >= 10:
    count = count + 1
    inp = inp - 10
while inp >= 5:
    count = count + 1
    inp = inp - 5
while inp >= 1:
    count = count + 1
    inp = inp - 1
print(count)

